Front avdo mmm

LamerMMM opened the pull request.

sraww committed:
Create README.md

Commit:
Test

LamerMMM committed:
Merge branch 'NRR1:ORG' into FrontAvdoMMM

Commit:
Перемещено Index от Medicine(view)
сделан шаблонный контроллер с 5 методами(включая get и post где возможно)
Исправлено наименование vm
косметические измения program.cs

Commit:
Долой JS пизуши!

sraww commented:
[appsettings.Development.json](https://github.com/user-attachments/files/20147483/appsettings.Development.json)
[appsettings.json](https://github.com/user-attachments/files/20147484/appsettings.json)
[README.md](https://github.com/user-attachments/files/20147485/README.md)

Commit:
Поправил layout, теперь он менее кривой
Перенес часть кода по папкам, вроде теперь должно быть менее криво(промежуточный)

Commit:
.
